apps/api/src/database/migrations/20230619130451_project_updates_sent/migration.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN TRY
```

Comment:
Is it worth just having 1 migration file for this or is the history useful?

Reply:
I don't think this kind of history is useful, combined: https://github.com/Planning-Inspectorate/back-office/pull/992/commits/a5efdba0d29a20661f82c69a64bb381a01dec384